docs/articles/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.md

```diff
@@ -77,22 +77,16 @@ To tell npm how to run your tests, you need to add the `test` script to the proj
 
 For more information on how to configure a test run using a `testcafe` command, see [Command Line Interface](../using-testcafe/command-line-interface.md).
 
-**Important:** If you are going to run tests for a website that is not deployed, the `test` script should also include commands to run and stop the site; for example:
+**Important:** If you are going to run tests for a website that is not deployed, the `test` script should also include commands to run the site. 
+Use the `--app` TestCafe option to specify a command that deploys the website locally.
+This command will be automatically executed before running tests. After tests are finished, TestCafe will stop the server.
 
 ```text
 "scripts": {
-  "test":  "node server.js & testcafe 'saucelabs:Chrome@beta:Windows 10' tests/index-test.js; EXIT_CODE=$?; kill $!; exit $EXIT_CODE"
+  "test":  "testcafe saucelabs:Chrome@beta tests/index-test.js --app 'node server.js'"
```

Comment:
Here and in other places don't use `'` in npm commands, it's interpreted incorrectly (at least on Windows), use escaped `\"` instead:
```json
"test":  "testcafe saucelabs:Chrome@beta tests/index-test.js --app \"node server.js\""
```